core/src/tangram.cpp

```diff
@@ -63,19 +62,35 @@ void initialize(const char* _scenePath) {
         logMsg("Loading Tangram scene file: %s\n", sceneRelPath.c_str());
         auto sceneString = stringFromResource(sceneRelPath.c_str());
 
-        SceneLoader loader;
+        if (SceneLoader::loadScene(sceneString, *m_scene)) {
+            m_tileManager->setScene(m_scene);
 
-        loader.loadScene(sceneString, *m_scene, *m_tileManager, *m_view);
+            glm::dvec2 projPos = m_view->getMapProjection().LonLatToMeters(m_scene->startPosition);
+            m_view->setPosition(projPos.x, projPos.y);
+            m_view->setZoom(m_scene->startZoom);
+        }
 
         m_skybox = std::unique_ptr<Skybox>(new Skybox("img/cubemap.png"));
         m_skybox->init();
-
     }
 
     logMsg("finish initialize\n");
 
 }
 
+void loadScene(const char* _scenePath) {
+    logMsg("Loading scene file: %s\n", _scenePath);
+
+    auto sceneRelPath = setResourceRoot(_scenePath);
+    auto sceneString = stringFromResource(_scenePath);
```

Comment:
`stringFromResource` needs to be called on `sceneRelPath` because after `setResourceRoot` is called, all resources will be resolved relative to the containing directory of `_scenePath`; this doesn't matter for the default location but for an absolute path like `/blah/scene.yaml` the second call will try to access `/blah//blah/scene.yaml`.